### PR TITLE
[HDRP] Area Lights: reduce static C# data size and faster LUT texture initialization

### PR DESCRIPTION
**Pull request by aras-p**
### Purpose of this PR

TL;DR: 2.5MB C# memory savings and faster area light lookup texture initialization. No change in rendering whatsoever.

For the 10 BRDFs possible in the area lights, there were pregenerated C# arrays of 64x64x9 doubles (9 doubles in each row, i.e. a 3x3 matrix).

All the code ever using them was only using 4 of those 9 doubles (0, 2, 4, 6 indices), and was always converting them to FP16 eventually - first into floats via SetPixels, then inside that into final FP16 texture format.

So skip all that, and instead generate 64x64x4 FP16 tables directly. These can also be set onto texture much more efficiently with a single SetPixelData call.

Reduces static C# array size of all of that from 2880KB to 320KB.

### Comments to reviewers

The AxF LTC area light code was updated too, but it _feels_ a bit WIP/unfinished. As in, it does create a LUT texture array with 3 elements, only ever initializes one element, and then proceeds to not use that LUT texture anywhere at all? The data for initializing it was different from other LTC LUTs (i.e. not normalized), but the upload code was still treating it the same.

### CODE REVIEW

**Comment by sebastienlagarde** [comment]
Thanks Aras for pointing this. We will evaluate the impact of the change (as you point in a comment some value differ) as we know LTC code is very sensitive to the fitted value, but anyway the change make a lot of sense.

**Comment by aras-p** [comment]
> as you point in a comment some value differ

The values do not differ. As in, with this change the rendering should be 100% identical to before. My comment was saying that AxF data tables "feel strange" -- I don't know how they are generated (whereas for regular LTC there's a utility class that rebuilds them), and the whole AxF LTC code "feels strange". That is kinda orthogonal to the change itself though.

**Comment by sebastienlagarde** [comment]
PR have been recreated in our repo. Closing this one. Thanks for reporting this.
